src/explore-education-statistics-frontend/src/prototypes/components/PrototypeSearchForm.tsx

```diff
@@ -151,6 +191,16 @@ class PrototypeSearchForm extends Component<{}, State> {
 
       this.setState({ currentlyHighlighted });
     }
+
+    if (e.keyCode === 13) {
```

Comment:
Use `e.key` as this is more ergonomic/readable than `e.keyCode` numbers

Reply:
String comparisons though?